sidebars.ts

```diff
@@ -53,6 +53,14 @@ const sidebars: SidebarsConfig = {
     },
```

Comment:
I am not sure whether it is a bug of this PR or it was before, but it is impossible to collapse L2 article lists, e.g. the "On-Chain Protocol" one. Upon clicking on the collapse arrow (\/), I am redirected to the "On-Chain Protocol" page.

Reply:
Fixed this
Non-collapsibility was forced by a block in custom.css
And the auto-generated index you got re-directed to upon clicking was my addition, I fixed both of them :D
[60c17be9f05f54649ab1c1d8cd0c8b2664ac7c0f](https://github.com/layer-3/docs/pull/69/commits/60c17be9f05f54649ab1c1d8cd0c8b2664ac7c0f)